### PR TITLE
Lashof cleanup

### PR DESCRIPTION
**Pull request by freyac**
Resolved issue #17 and updated example notebook.

### CODE REVIEW

**Comment by freyac** [comment]
@jhamman – can we check base sometime soon to resolve the failing test?

**Comment by jhamman** [comment]
@freyac - I don't think there's anything wrong here. If this is all done, I think we can merge as is.

File: tonyear/core.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def calculate_tonyears(
     Parameters
```
Comment (badgley) [inline tonyear/core.py:90]: I think it's worthwhile adding a note of caution about the `discount_rate` parameter and maybe pointing the reader to the docs. Definitely don't have to get into it too much but maybe something like:

"Extreme caution should be used when discounting IRF curves. See documentation."

File: notebooks/ton-year.ipynb [hunk 2]
```diff
@@ -12,25 +12,31 @@
...
   {
    "cell_type": "markdown",
-   "id": "90656242",
+   "id": "39a749c5",
    "metadata": {},
    "source": [
-    "This section models two Impulse Reponse Functions (IRFs). An IRF represents the\n",
-    "time-dependent fraction of an emission pulse remaining in the atmosphere at time\n",
-    "t. It is usually approximated by a sum of exponentials:\n",
+    "An Impulse Reponse Functions (IRFs) represents the time-dependent fraction of an\n",
```
Comment (badgley) [inline notebooks/ton-year.ipynb:37]: "An impulse response function (IRF)"

File: notebooks/ton-year.ipynb [hunk 3]
```diff
@@ -123,194 +89,79 @@
...
-    "        # The Moura-Costa method calculates the ton-year benefit of a delayed emission\n",
-    "        # as the ton-years of carbon storage outside of the atmosphere over the period\n",
-    "        # 0<=t<=delay. Moura-Costa ignores the atmospheric impact of post-storage re-emission.\n",
-    "        delay_timesteps = delay + 1\n",
-    "        scenario = [-1] * (delay_timesteps) + [0] * (\n",
-    "            len(baseline) - delay_timesteps\n",
-    "        )\n",
-    "        scenario = get_discounted_curve(discount_rate, scenario)\n",
-    "        benefit = -np.trapz(scenario[:delay_timesteps])\n",
+    "### Parameters\n",
```
Comment (badgley) [inline notebooks/ton-year.ipynb:107]: Would be nice to point out that we have an 'overview' in the docs where we discuss the parameters (actually we even discuss the IRF curves too!) in more depth.

**Review by badgley — COMMENTED** [review]
looks great! 

:shipit:

**Pull request merged**